time/src/offset_date_time.rs

```diff
@@ -1596,6 +1596,7 @@ impl From<js_sys::Date> for OffsetDateTime {
     ///
     /// This may panic if the timestamp can not be represented.
     fn from(js_date: js_sys::Date) -> Self {
+        use time_core::convert::Millisecond;
```

Comment:
I would prefer these imports to be at the top of the file. It would be reasonable to do `use crate::convert::*;` to avoid spelling everything out and dealing with gating it all correctly.